designs/administrator/olm/operand-details-grouping/index.md

```diff
@@ -0,0 +1,36 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Improve grouping and editing on Operand Details view
+
+When creating an operand, fairly complex nesting and hierarchy of fields is used to set values on the operand. Once created however, the Details tab for that operand does not show this hierarchy and all fields are shown at the same flat level, with somewhat of a mix of spec and status descriptors, with some hierarchical section headers missing as well. This can make this view challenging to interpret.
+
+This design describes surfacing the common spec descriptors first, the status descriptors next, then the rest of the spec descriptors, with a divider separating them. Groups of spec descriptors will now have their grouping reflected using a box for the first level, with level indicator labels conveying any further hierarchy. An option to edit these groups using their existing form view will also be available.
+
+## Separating Spec and Status descriptors
+
+![Dividing groups](img/8-1-divider.png)
+- A new divider will be added between the status and spec descriptor sections to visually separate them.
+
+## Grouping Spec descriptors
+
+![Conveying groups](img/9-1-group.png)
+- The top level of spec descriptors' hierarchy will now be shown grouped in a box.
+- Each field has a level indicator to exactly convey its hierarchy, which would be important for fields beyond the first depth level.
+
+![Conveying groups popover](img/9-2-groupPop.png)
+- Each descriptor will support an existing popover style that shows more information on the header click, including the description and full hierarchy level path, useful if the level indicator is truncated. 
+
+## Allowing quick access to edit form from a Spec group
+
+![Group editing](img/7-1-editGroup.png)
+- Each top level group will include an **Edit** link, allowing the user to quickly view the full form view of that group, and edit all of its fields.
+
+![Group editing modal](img/7-3-modal.png)
```

Comment:
I really like this visual treatment and wonder if it's something we should reuse on the main form to show a grouping. I guess the Edit link might feel like it is floating a bit. Maybe something @mceledonia could take a look at if he hasn't already?

Reply:
Agreed! Mentioned in the PF issue tracking grouping: https://github.com/patternfly/patternfly-design/issues/889

Reply:
Just a nit: For the accordion component any expanded section has a blue heading
https://www.patternfly.org/v4/documentation/react/components/accordion#fixed-with-multiple-expand-behavior
and it looks like the left border extends up beyond the title of the section, so here you'd have a blue border without any break.

Reply:
Interestingly that was a screenshot from the current form, but I have updated the image to reflect how the component should look, thanks.

Reply:
@matthewcarleton @itsptk We had to make some adjustments to the PF accordion for the form view. The PF accordion component style adds left padding to the accordion toggle, which causes it not to align with the rest of the normal form field labels:

![pf accordion indention](https://user-images.githubusercontent.com/22625502/88114748-95f0e180-cb82-11ea-8e74-fe10919bab74.gif)

Reply:
ah ok thanks for the clarification @TheRealJon 
Should this be something that goes back to PF? Should we keep the blue text for expanded sections?

Reply:
Just opened https://github.com/patternfly/patternfly-design/issues/904 to ask about the padding to the left of the Accordion.

The headings are now blue when expanded, so I think we are good.